app/views/gobierto_plans/plans/show.html.erb

```diff
@@ -2,16 +2,22 @@
   <%= link_to title(@plan.title), gobierto_plans_plan_path(@plan.slug) %>
 <% end %>
 
+<% if @plan.css.present? %>
+  <style type='text/css'>
+    <%== @plan.css %>
+  </style>
+<% end %>
+
 <div class="column">
   <div class="block">
 
     <div class="pure-g header_block_inline m_b_1">
       <div class="pure-u-1 pure-u-md-12-24">
         <h2 class="with_description">
-          <%= t(".title") %>
+          <%= @plan.title %>
         </h2>
         <p>
-          <%= t(".description") %>
+          <%= @plan.introduction %>
         </p>
         <p>
           <%= t(".latest_update_html", date: l(@plan.updated_at.to_date, format: :long)) %>
```

Comment:
Instead of the `plan.updated_at`, use the activity `gobierto_plans.plan_type_updated`. That activity should be generated when the plans are updated in the API script. Check how is done for budgets.